spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'spec_helper'
+
+describe ApplicationHelper do
+  include ApplicationHelper
+
+  context '#can_see_full_notice_version?' do
+    let(:notice) { create(:dmca) }
+    let(:token_url) do
+      create(
+        :token_url,
+        email: 'user@example.com',
+        notice: notice
+      )
+    end
+
+    it 'returns true when the user is allowed to see the full notice version' do
```

Comment:
Does (and should) it still work when a logged-in user with permissions happens to be using an invalid url?

Reply:
> Does (and should)

Yes (and yes).

Reply:
Cool. Please add a test for that case.

Reply:
Done!